apps/spruce/src/pages/waterfall/InactiveVersionsButton/index.tsx

```diff
@@ -35,6 +41,12 @@ export const InactiveVersionsButton: React.FC<Props> = ({
           />
         ))}
       </DisplayModal>
+      {brokenVersionsCount > 0 && (
+        <StyledBadge data-cy="broken-versions-badge" variant={Variant.Red}>
+          {brokenVersionsCount} broken{" "}
+          {pluralize("version", brokenVersionsCount)}
```

Comment:
Sorry for not suggesting this earlier, but what do you think of omitting "versions" entirely? I don't think it's so helpful that it warrants taking away space from the useful columns on the page. Without it we could just have one fixed width for the `InactiveVerison` div I think.